### PR TITLE
VPN-5969: Update settings screen

### PR DESCRIPTION
**Pull request by mcleinman**
## Description

Updates Settings screens per written details and mocks on ticket. 

I've kept the string ids the same as we'd rather keep the existing translations than have no translation for a bit - and let them slowly get adjusted as needed.

(I touched some test files, and the formatting got patched up - there aren't significant changes on most of those lines that just lose some whitespace.)

## Reference

VPN-5969

## Checklist
    
- [x] My code follows the style guidelines for this project
- [x] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [x] I have performed a self review of my own code
- [x] I have commented my code PARTICULARLY in hard to understand areas
- [x] I have added thorough tests where needed

### CODE REVIEW

File: addons/message_upgrade_to_annual_plan/manifest.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
         "id": "c_4",
         "type": "button",
         "style": "link",
-        "content": "Get help",
+        "content": "Help",
```
Comment (flodolo) [inline addons/message_upgrade_to_annual_plan/manifest.json:52]: This needs a new ID.

**Review by flodolo — APPROVED** [review]
Every string change requires a new ID (addons, strings.yaml)

**Review by flodolo — CHANGES_REQUESTED** [review]
(wrong button 🙄)

File: addons/message_upgrade_to_annual_plan/manifest.json (hunk 1 above)
Comment (mcleinman) [reply addons/message_upgrade_to_annual_plan/manifest.json:52]: Fixed - changed everything to having new IDs

**Review by lesleyjanenorton — APPROVED** [review]
Looks great!

File: src/ui/screens/devices/ViewDeviceLimit.qml [hunk 2]
```diff
@@ -19,15 +19,15 @@ MZViewBase {
...
             onClicked: {
                 Glean.sample.getHelpClickedInitialize.record();
                 MZNavigator.requestScreen(VPN.ScreenGetHelp);
             }
         }
     }
 
-    //% "My devices"
+    //% "Devices"
     _menuTitle: qsTrId("vpn.devices.myDevices")
```
Comment (flodolo) [inline src/ui/screens/devices/ViewDeviceLimit.qml:31]: Realized only today that this and another string were changed without a new ID, so localization will reflect the English text only from the next version of VPN 😞

**Pull request merged**